From cce036dad0395b49d1aadf80a93688ff0d1673af Mon Sep 17 00:00:00 2001
From: Guillaume Papin <guillaume.papin@epitech.eu>
Date: Mon, 30 Apr 2018 01:06:11 +0200
Subject: [PATCH 1/7] turn off libxml2 as it is used only for testing or
 windows code

Only the following files seems to use libxml:
- tools/clang/tools/c-index-test/c-index-test.c
- lib/WindowsManifest/WindowsManifestMerger.cpp
---
 cmake/config-ix.cmake | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/cmake/config-ix.cmake b/cmake/config-ix.cmake
index aaf22ff..3c2251c 100644
--- a/cmake/config-ix.cmake
+++ b/cmake/config-ix.cmake
@@ -169,14 +169,6 @@ if(NOT LLVM_USE_SANITIZER MATCHES "Memory.*")
     find_library(ICONV_LIBRARY_PATH NAMES iconv libiconv libiconv-2 c)
     set(LLVM_LIBXML2_ENABLED 0)
     set(LIBXML2_FOUND 0)
-    if((LLVM_ENABLE_LIBXML2) AND ((CMAKE_SYSTEM_NAME MATCHES "Linux") AND (ICONV_LIBRARY_PATH) OR APPLE))
-      find_package(LibXml2)
-      if (LIBXML2_FOUND)
-        set(LLVM_LIBXML2_ENABLED 1)
-        include_directories(${LIBXML2_INCLUDE_DIR})
-        set(LIBXML2_LIBS "xml2")
-      endif()
-    endif()
   endif()
 endif()
 
-- 
2.10.0

